apps/prairielearn/python/zygote.py

```diff
@@ -216,6 +223,28 @@ def worker_loop() -> None:
                 # fast as possible.
                 os._exit(0)
 
+            if fcn is None:
+                sys.stderr.write("fcn is required and is None")
+                continue
+
+            if file is None:
+                sys.stderr.write(f"file is required for fcn type {fcn}, and is None")
+                continue
+
+            if args is None or len(args) == 0:
+                sys.stderr.write(
+                    f"args is required for fcn type {fcn}, and is None or empty"
+                )
+                continue
+
+            if cwd is None:
+                sys.stderr.write(f"cwd is required for fcn type {fcn}, and is None")
+                continue
+
+            if not is_phase(fcn):
```

Comment:
This is a potential breakage: invalid fcn values that were unknown were previously ignored.